tests/e2e/cluster-scoped/cluster_scoped_rollouts_test.go

```diff
@@ -0,0 +1,330 @@
+			Eventually(rolloutsManager, "1m", "1s").Should(rmFixture.HaveCondition(
+				metav1.Condition{
+					Type:    rmv1alpha1.RolloutManagerConditionType,
+					Status:  metav1.ConditionTrue,
+					Reason:  rmv1alpha1.RolloutManagerReasonSuccess,
+					Message: "",
+				}))
+
+			By("Verify that expected resources are created.")
+			utils.ValidateArgoRolloutManagerResources(ctx, rolloutsManager, k8sClient, false)
+
+			By("Verify argo Rollouts controller is able to reconcile CR of other namespace.")
+
+			By("Create a different namespace for Rollout.")
+			Expect(utils.CreateNamespace(ctx, k8sClient, nsName2)).To(Succeed())
+
+			By("Create and validate Rollout.")
+			utils.ValidateArgoRolloutsResources(ctx, k8sClient, nsName2, 31000, 32000)
+		})
+
+		/*
+			In this test a cluster-scoped RolloutManager is created in a namespace.
+			Then operator should create required resources (ServiceAccount, ClusterRoles, ClusterRoleBinding, Service, Secret, Deployment) in one namespace.
+			Now when a Rollouts CR is created in multiple namespaces, operator should still be able to reconcile all of them.
+		*/
+		It("After creating cluster-scoped RolloutManager in a namespace, operator should create appropriate K8s resources and watch argo rollouts CR in multiple namespace.", func() {
+
+			nsName1, nsName2, nsName3 := "rom-ns-1", "ro-ns-1", "ro-ns-2"
+
+			By("Create a namespace for RolloutManager.")
+			Expect(utils.CreateNamespace(ctx, k8sClient, nsName1)).To(Succeed())
+
+			By("Create cluster-scoped RolloutManager.")
+			rolloutsManager, err := utils.CreateRolloutManager(ctx, k8sClient, "test-rollouts-manager-1", nsName1, false)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Verify that RolloutManager is successfully created.")
+			Eventually(rolloutsManager, "1m", "1s").Should(rmFixture.HavePhase(rmv1alpha1.PhaseAvailable))
+
+			By("Verify that Status.Condition is set.")
+			Eventually(rolloutsManager, "1m", "1s").Should(rmFixture.HaveCondition(
+				metav1.Condition{
+					Type:    rmv1alpha1.RolloutManagerConditionType,
+					Status:  metav1.ConditionTrue,
+					Reason:  rmv1alpha1.RolloutManagerReasonSuccess,
+					Message: "",
+				}))
+
+			By("Verify that expected resources are created.")
+			utils.ValidateArgoRolloutManagerResources(ctx, rolloutsManager, k8sClient, false)
+
+			By("Verify argo Rollouts controller is able to reconcile CR of multiple namespaces.")
+
+			By("1st Namespace: Create a different namespace for Rollout.")
+			Expect(utils.CreateNamespace(ctx, k8sClient, nsName2)).To(Succeed())
+
+			By("1st Namespace: Create and validate Rollout in 1st namespace.")
+			utils.ValidateArgoRolloutsResources(ctx, k8sClient, nsName2, 31000, 32000)
+
+			By("2nd Namespace: Create a another namespace for 2nd Rollout.")
+			Expect(utils.CreateNamespace(ctx, k8sClient, nsName3)).To(Succeed())
+
+			By("2nd Namespace: Create and validate Rollout in 2nd namespace.")
+			utils.ValidateArgoRolloutsResources(ctx, k8sClient, nsName3, 31001, 32002)
```

Comment:
Actually these are two different scenarios, in 1st test case, it is checking that 2 cluster scope RMs should not be allowed and and in 2nd it is checking that namespace scoped RM should not be allowed if a cluster scoped RM is available. Both could be merged but that it will be little confusing, I think.